src/main/javascript/src/pages/role/update/update.js

```diff
@@ -14,7 +14,7 @@ export class updateRoleFormFields {
         this.selectedPrivileges = [];
     }
 
```

Comment:
In this class, theres no feedback on why we can't delete roles. 
![image](https://user-images.githubusercontent.com/32455969/78314112-f3efaf80-7526-11ea-9c12-5b00fada448b.png)
When updating the role privileges I didnt expect all of them to be deleted when I say to update them to this one role, but I suppose that's more my problem than the code.
Doing so also makes a lot of duplicates because update and create are not different enough. 
![image](https://user-images.githubusercontent.com/32455969/78314293-87c17b80-7527-11ea-8bba-adb7a25ce577.png)